### DISCUSSION TITLE
Did the devs remove the flag related to the close all tabs dialog?

### DISCUSSION THREAD

**Discussion by PassionateG1t** [bromite/bromite#2231] [Q&A]
There used to be a flag concerning this that when enabled you wouldn't need to confirm closing all tabs. It's no longer available now but opening an issue would be automatically discarded as it's not privacy related. It might be upstream but I think bromite devs can reimplement it iirc.

**Comment by csagan5** [comment]
It was removed upstream in https://github.com/chromium/chromium/commit/614a998c5af5dcd36baffa6d65090c0b8decb03f

I will try to re-introduce it in next version.
